### PR TITLE
Add instructions for linting.

### PR DESCRIPTION
**Pull request by briantkennedy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,20 @@ Replaced:
...
+go get github.com/forseti-security/config-validator/cmd/policy-tool
+policy-tool --policies ./policies --policies ./samples --libs ./lib
+```
+
+### Local CI
+You can run the cloudbuild CI locally as follows:
+
+```
+gcloud components install cloud-build-local
+cloud-build-local --config build/cloudbuild/test-cloudbuild.yaml --dryrun=false .
```
Comment (morgante) [inline README.md]: I don't see this file?

File: README.md [hunk 2]
```diff
@@ -42,3 +42,20 @@ Replaced:
 #contents of my_rule.rego
 #ENDINLINE
 ```
+
+### Linting Policies
+FCV provides a policy linter.  You can invoke it as:
+
+```
+go get github.com/forseti-security/config-validator/cmd/policy-tool
```
Comment (morgante) [inline README.md]: Note it'd likely be good for us to upload a copy to GCS somewhere potentially.

Comment (briantkennedy) [reply README.md]: The eventual plan is to add this check as a cloudbuild step once the linter has a Dockerfile (sometime this week).

File: README.md (hunk 1 above)
Comment (briantkennedy) [reply README.md]: got mixed up with FCV, fixed

**Pull request merged**